spec/chitter_db_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'peep'
+
+describe Peep do
+  describe '.all' do
+    it 'returns all peeps' do
+      connection = PG.connect(dbname: 'chitter_test')
+
+      peep = Peep.create(body: 'My first peep!', user: '@indisaurusrex')
```

Comment:
You dont need the "peep =" as you do not do anything with the variable peep after it is assigned and do not return anything from the create method.